br/OWNERS

```diff
@@ -1,10 +1,5 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 options:
   no_parent_owners: true
-filters:
-  "(tidb-lightning\\.toml)$":
-    approvers:
-      - sig-critical-approvers-tidb-lightning
```

Comment:
@lyzx2001 you forgot to move this part into `lightning/OWNERS`.

Reply:
done
https://github.com/pingcap/tidb/pull/52745